va-gov/layouts/home.html

```diff
@@ -223,4 +223,19 @@ <h4 class="homepage-news-story-title"><a href=https://www.blogs.va.gov/VAntage/4
 
 </main>
 
```

Comment:
Let me know if this is an okay approach for. I felt like this would be better then just putting an onClick event onto the element, but I can change it if it makes more sense the original way we do it.